pkg/apis/batch/validation/validation_test.go

```diff
@@ -1349,6 +1373,46 @@ func TestValidateJobUpdate(t *testing.T) {
 				job.Spec.ManualSelector = pointer.Bool(true)
 			},
 		},
+		"invalid attempt to set managedBy field": {
+			old: batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "abc",
+					Namespace: metav1.NamespaceDefault,
+					Labels:    map[string]string{},
```

Comment:
```suggestion
				ObjectMeta: validJobObjectMeta,
```

We already wouldn't care about labels.

Reply:
done, use the oneliner as in other test cases